packages/shared/src/forEachAction.ts

```diff
@@ -155,11 +155,10 @@ const forEachActionRecur = (
           replaceOrDeleteActions(doneTransition, 'actions', visitor);
         }
       }
-    }
-    /**
-     * invoke.onError transitions
-     */
-    if (event.startsWith('error.invoke')) {
+    } else if (event.startsWith('error.invoke')) {
```

Comment:
Should this be `error.platform`?

Reply:
It kinda should but we are also not quite relying on this here anyway. A better "fix" for this would be to just manipulate the config directly anyway so I decided not to touch this for the time being.